packages/text-bitmap/src/utils/splitTextToCharacters.ts

```diff
@@ -0,0 +1,4 @@
+export function splitTextToCharacters(text: string): string[]
```

Comment:
```suggestion
/**
 * Ponyfill for IE because it doesn't support `Array.from`
 * @private
 */
export function splitTextToCharacters(text: string): string[]
```